mmf/utils/distributed.py

```diff
@@ -169,6 +200,12 @@ def byte_tensor_to_object(byte_tensor, max_size=4094):
 def infer_init_method(config):
     if config.distributed.init_method is not None:
         return
+
+    if getattr(config.training, "device", "cuda") == "xla":
+        registry.register("is_xla", True)
+    else:
+        registry.register("is_xla", False)
```

Comment:
```suggestion
    registry.register("is_xla", config.training.get("device", "cuda") == "xla")
```

Reply:
I agree. Much better. Thanks for suggestion. Just committed your proposed change.